### PR TITLE
Add AWS WAF Plugin

### PR DESCRIPTION
**Pull request by holidayworking**
I created AWS WAF Plugin.

### CODE REVIEW

**Comment by stefafafan** [comment]
@holidayworking 

Thank you for the pull request. Overall it looked nice.

One point I noticed is that this plugin only supports AWS WAF for CloudFront, and not the metrics of WAF for ALB. I thought it may be nice to mention this in the README document.

File: mackerel-plugin-aws-waf/main.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/holidayworking/mackerel-plugin-aws-waf/lib"
```
Comment (stefafafan) [inline mackerel-plugin-aws-waf/main.go]: The import path is pointing to your repository, this would need to be fixed as well.

Comment (holidayworking) [inline mackerel-plugin-aws-waf/main.go]: Fix cb6f0e7

**Comment by holidayworking** [comment]
@stefafafan 

> One point I noticed is that this plugin only supports AWS WAF for CloudFront, and not the metrics of > WAF for ALB. I thought it may be nice to mention this in the README document.

I add to mention this in the README document.

**Comment by stefafafan** [comment]
@holidayworking 
Thank you for the fixes, LGTM!

**Comment by holidayworking** [comment]
@stefafafan Thank you!

**Pull request merged**